devise.rb

```diff
@@ -4,23 +4,24 @@
 ########################################
 inject_into_file 'Gemfile', before: 'group :development, :test do' do
   <<~RUBY
-    gem 'devise'
-
     gem 'autoprefixer-rails', '10.2.5'
+    gem 'devise'
     gem 'font-awesome-sass'
     gem 'simple_form'
```

Comment:
```suggestion
    gem 'simple_form'
    
```

We could add a return line here for the gems to be separated from the next line `group :development, :test do`